ivy/container/manipulation.py

```diff
@@ -333,6 +333,22 @@ def constant_pad(
         map_sequences: bool = False,
         out: Optional[ivy.Container] = None,
     ) -> ivy.Container:
+        """
+        ivy.Container instance method variant of ivy.constant_pad. This method simply wraps the
+        function, and so the docstring for ivy.constant_pad also applies to this method
+        with minimal changes.
+
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([1, 2, 3]), \
+                              b=ivy.array([4, 5, 6]))
+        >>> y = x.constant_pad((2, 3), (5, 6))
+        >>> print(y)
+        {
+            a: ivy.array([5, 5, 1, 2, 3, 6, 6, 6,]),
```

Comment:
Hi, so the ```,``` at the end, inside this array, is causing the tests to fail. Can you please remove it?